config.yaml

```diff
@@ -2,7 +2,10 @@ debug: true # Enable debug mode (more logs)
 hooks: # Enabled hooks
   - acl
   - removeContainer
+  - insertIptables
+  - removeIptables
 proc_path: /mnt/proc # Mount path of host proc folder
+container_bridge_interface: docker0
```

Comment:
Maybe use embedded hook conf (for readability):

iptables:
  bridge_interface: docker0